include/types/Profiles.h

```diff
@@ -13,12 +13,16 @@
 class Profiles : public PerformanceTable {
 public:
   /**
-   * Profiles standard constructor (PerformanceTable surcharged)
+   * Profiles standard constructor (PerformanceTable surcharged). A Profile type
+   * is always represented in the mode_ = "crit" of a PerformanceTable. There is
+   * no such thing as a Profile in "alt" mode.
```

Comment:
Is this still true (only one mode for profile)? 
If so, why do we have the choice of the mode for the profile?